python/tvm/relay/op/contrib/ethosn.py

```diff
@@ -178,18 +186,29 @@ def check_avg_pool2d(extract):
 
         return support.avg_pool2d(extract)
 
+    def check_mean(extract):
+        """Check if mean is supported by Ethos-N."""
+        if not ethosn_available():
+            return False
+
+        return support.mean(extract)
+
     def check_sigmoid(extract):
         """Check if a sigmoid is supported by Ethos-N."""
         if not ethosn_available():
             return False
 
+        if extract.attrs.out_dtype != "uint8":
+            return False
+
```

Comment:
This should not be here because the int8 support for sigmoid has been merged earlier this week.